src/inspector_socket.h

```diff
@@ -58,7 +59,7 @@ class InspectorSocket {
   struct http_parsing_state_s* http_parsing_state;
   struct ws_state_s* ws_state;
   std::vector<char> buffer;
-  uv_tcp_t client;
+  uv_tcp_t client;  // XXX(sam) this is a server, why is it called "client"?
```

Comment:
I found this confusing, it is a TCP connection, and it is connected to a "client" on the other side, but this side is the "server" side of the HTTP/Websocket connection. I'd suggest just calling it "tcp" (because it is a `uv_tcp_t`) or "socket" (because it wraps a socket).